src/main/resources/de/tracetronic/jenkins/plugins/ecutestexecution/steps/ProvideLogsStep/config.properties

```diff
@@ -0,0 +1,2 @@
+step.description=This step provides all ecu.test logs in the "ecu.test logs" tab and as artifacts in jenkins.
```

Comment:
This step provides all ecu.test logs ~in the "ecu.test logs" tab and~ as job artifacts ~in jenkins~.

Btw ~j~Jenkins always capitalized